script.js

```diff
@@ -0,0 +1,15 @@
+// Open mobile popup menu
+function displayPopUp(){
+ const pop_up_menu=document.querySelector("#popup-menu-id");
+ pop_up_menu.style.display="block";
+ pop_up_menu.style.right=0;
+}
+
+// close mobile popup menu
+function closePopUp(){
```

Comment:
1. When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed. ✅
2. When the user clicks (or taps) any option of the mobile menu, the mobile menu disappear. ❌

Your mobile menu is not closed when I click these options. Kindly make it work.